### PR TITLE
chore: refactor create event form 

### PR DESCRIPTION
**Pull request by Sboonny**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

this targets making the inputs more user friendly, I should replace foo bar from other forms but this for different PR

lint issues are taken care of by this PR https://github.com/freeCodeCamp/chapter/pull/1625/files, so I added blocked label until the first one is merged

### CODE REVIEW

File: cypress/e2e/dashboard/chapters/[chapterId]/chapterId-index.cy.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ describe('chapter dashboard', () => {
     Object.entries(testEvent).forEach(([key, value]) => {
       // TODO: simplify this conditional when tags and dates are handled
       // properly.
-      if (!['tags', 'start_at', 'ends_at', 'venue_id'].includes(key)) {
+      if (!['tags', 'startAt', 'endAt', 'venueId'].includes(key)) {
```
Comment (Sboonny) [inline cypress/e2e/dashboard/chapters/[chapterId]/chapterId-index.cy.ts:49]:
```suggestion
      if (!['tags', 'start_at', 'ends_at', 'venue_id'].includes(key)) {

File: cypress/e2e/dashboard/chapters/[chapterId]/chapterId-index.cy.ts [hunk 2]
```diff
@@ -109,24 +109,29 @@ describe('chapter dashboard', () => {
   function createEventViaUI(chapterId) {
     cy.visit(`/dashboard/chapters/${chapterId}`);
     cy.get(`a[href="/dashboard/chapters/${chapterId}/new-event"]`).click();
-    cy.findByRole('textbox', { name: 'Event title' }).type(testEvent.name);
+    // ToDo: textbox shouldn't use component's text to indicate if it can find the component.
```
Comment (ojeytonwilliams) [inline cypress/e2e/dashboard/chapters/[chapterId]/chapterId-index.cy.ts:112]:
```suggestion
```
We do need to improve the tests, but I'm not sure this is the way.

File: cypress/e2e/dashboard/chapters/[chapterId]/chapterId-index.cy.ts [hunk 3]
```diff
@@ -109,24 +109,29 @@ describe('chapter dashboard', () => {
...
+    cy.findByRole('textbox', { name: 'Event Title (Required)' }).type(
+      testEvent.name,
+    );
     cy.findByRole('textbox', { name: 'Description' }).type(
       testEvent.description,
     );
     cy.findByRole('textbox', { name: 'Event Image Url' }).type(
       testEvent.image_url,
     );
     // cy.findByRole('textbox', { name: 'Url' }).type(testEvent.url);
```
Comment (ojeytonwilliams) [inline cypress/e2e/dashboard/chapters/[chapterId]/chapterId-index.cy.ts:122]:
```suggestion
```

File: cypress/e2e/dashboard/events/events-index.cy.ts [hunk 4]
```diff
@@ -219,7 +218,10 @@ describe('spec needing owner', () => {
     cy.findByRole('link', { name: 'Edit' }).click();
     const titleAddon = ' new title';
 
-    cy.findByRole('textbox', { name: 'Event title' }).type(titleAddon);
+    // ToDo: textbox shouldn't use component's text to indicate if it can find the component.
```
Comment (ojeytonwilliams) [inline cypress/e2e/dashboard/events/events-index.cy.ts:221]:
```suggestion
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Now the tags are gone, those tests will need updating. That should be all, though!

**Comment by ojeytonwilliams** [comment]
Sorry for adding to your merge conflict burden :disappointed:

**Comment by Sboonny** [comment]
Yeab, I have no problems with conflicts, no worries 👍

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**